manifests/pipecd/values.yaml

```diff
@@ -83,6 +83,9 @@ secret:
   gcsServiceAccount:
     fileName: "gcs-service-account"
     data: ""
+  sqlServiceAccount:
```

Comment:
nit: `cloudSQLServiceAccount`? We're going to support the authentication for Amazon RDS, right?

Reply:
Well, this change is for authorization, not authentication. And I've not looked at Amazon RDS connect method yet, but basically, we only support authentication RDS by username/password as we do for now, this service account is specific for connect to GCP cloud SQL only 🙏 
About the name change, I will adopt your suggestion since it would be better to clearly mention that this is only for cloud SQL as you said 🙆‍♀️

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2361/commits/48309fffe4394196e14c266694924d9b0bc704f0 🙏